open-cluster-management/backlog#7709 clusterroles

cadawson opened the pull request.

cadawson committed:
open-cluster-management/backlog#7709 clusterroles

skeeey commented on manage_cluster/define_clusterrole.adoc:
```diff
@@ -0,0 +1,81 @@
+[#configuring-a-specific-cluster-management-role]
+= Configuring a specific cluster management role
+
+When you install {product-title}, the default configuration provides the `cluster-admin` role on the {product-title-short} hub cluster. This permission enables you to create, manage, and import managed clusters on the hub cluster. In some situations, you might want to limit the access to certain `ManagedClusterSets` that are managed by the hub cluster, rather than access to all of the managed clu ...
```
This should be `ManagedCluster` not `ManagedClusterSets`

skeeey commented on manage_cluster/define_clusterrole.adoc:
```diff
@@ -0,0 +1,81 @@
+
+When you install {product-title}, the default configuration provides the `cluster-admin` role on the {product-title-short} hub cluster. This permission enables you to create, manage, and import managed clusters on the hub cluster. In some situations, you might want to limit the access to certain `ManagedClusterSets` that are managed by the hub cluster, rather than access to all of the managed clu ...
+
+You can limit access to certain `ManagedClusterSets` by defining a `clusterrole` and applying it to a user or group. Complete the following steps to configure and apply a role:
```
This should be `ManagedCluster` not `ManagedClusterSets`

skeeey commented on manage_cluster/define_clusterrole.adoc:
```diff
@@ -0,0 +1,81 @@
+  +
+  Replace `_clusterrole-name_` with the name of the cluster role that you are creating.
+  +
+  Replace `_managed-cluster-name_` with the name of the managed cluster or `ManagedClusterSet` that you want the user to have access to. 
```
no `ManagedClusterSet `

dockerymick commented on manage_cluster/define_clusterrole.adoc:
```diff
@@ -0,0 +1,81 @@
+  +
+  Replace `_managed-cluster-name_` with the name of the managed cluster or `ManagedClusterSet` that you want the user to have access to. 
+  
+  . Apply the clusterrole definition by entering the following command:
```
Can you add code back-ticks around clusterrole

cadawson committed:
open-cluster-management/backlog#7709 rev and peer

cadawson commented:
Hello @skeeey . I addressed your comments in the document. I think we are ready to go. Would you like to take a final look at it and approve, please? Thank you!

cadawson committed:
Merge branch '2.2_stage' into dawson770922

skeeey commented:
done
